manifests/system_cpu.pp

```diff
@@ -0,0 +1,19 @@
+class puppet_metrics_collector::system_cpu (
+  Integer $collection_frequency      = $puppet_metrics_collector::system::collection_frequency,
+  Integer $polling_frequency_seconds = $puppet_metrics_collector::system::polling_frequency_seconds,
+  Integer $retention_days            = $puppet_metrics_collector::system::retention_days,
+  String  $metrics_ensure            = $puppet_metrics_collector::system::system_metrics_ensure,
+  ) {
+  Puppet_metrics_collector::Sar_metric {
+    output_dir                => $puppet_metrics_collector::system::output_dir,
+    scripts_dir               => $puppet_metrics_collector::system::scripts_dir,
+    cron_minute               => "*/${collection_frequency}",
+    collection_frequency      => $collection_frequency,
+    polling_frequency_seconds => $polling_frequency_seconds,
+    retention_days            => $retention_days,
+  }
+
+  puppet_metrics_collector::sar_metric { 'cpu' :
+    metric_ensure             => $metrics_ensure,
```

Comment:
This is really a comment on the `generate_system_metrics`, but since that has already been merged, I am putting it here. 

`generate_system_metrics` does not test to ensure that the `sar` binary is installed on the system.

```
[root@pe-201921-master scripts]# /opt/puppetlabs/puppet-metrics-collector/scripts/generate_system_metrics --metric_type cpu --file_interval 300 --polling_interval 1 --metrics_dir /opt/puppetlabs/puppet-metrics-collector
Traceback (most recent call last):
        3: from /opt/puppetlabs/puppet-metrics-collector/scripts/generate_system_metrics:255:in `<main>'
        2: from /opt/puppetlabs/puppet-metrics-collector/scripts/generate_system_metrics:157:in `generate_system_metrics'
        1: from /opt/puppetlabs/puppet-metrics-collector/scripts/generate_system_metrics:71:in `run_sar'
/opt/puppetlabs/puppet-metrics-collector/scripts/generate_system_metrics:71:in ``': No such file or directory - sar (Errno::ENOENT)
```

We should consider testing for the executable and writing out an easy to read error message to the files if the binary is not executable by the script.

Reply:
created SLV-707 for to fix this